### DISCUSSION TITLE
Video stuck in firefox

### DISCUSSION THREAD

**Discussion by vishnukum-code** [vimeo/player.js#1011] [Bug reports]
Video doesn't play in firefox. But in other browser the video is playing perfectly. An error occurs in the console which is ("Uncaught (in promise) DOMException: The fetching process for the media resource was aborted by the user agent at the user's request.")

![image_2021_09_17T08_48_43_416Z](https://user-images.githubusercontent.com/77272103/133760474-245597d6-c4bd-46b7-972d-334c6a4b38f0.png)

**Comment by lssamlyon** [comment]
Thanks for reporting this to us. 

Can we get some more details on the page you are using and how you have the player integrated into the page?  If you can provide a link to the page that would assist us greatly.  We don't quite have enough detail in this report to understand whats breaking and where,  so please follow up with more details and I can create a ticket with the team internally.

**Comment by bdougherty** [comment]
Nothing we can really do here without more details. Besides that, it doesn't seem related to the player API or API library.
